src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const filteredStylesList = sourceString
+    .split(';')
+    .filter((style) => /\S/.test(style));
+  const resultObject = {};
+
+  for (const styleItem of filteredStylesList) {
+    const keyValue = styleItem.split(':').map((item) => item.trim());
+    const [property, value] = [...keyValue];
```

Comment:
The use of the spread operator `[...keyValue]` is unnecessary here since `keyValue` is already an array that can be destructured directly into `property` and `value`. You can simplify this line to `const [property, value] = keyValue;`.